column.go

```diff
@@ -101,7 +101,6 @@ func NewColumn(h api.SQLHSTMT, idx int) (Column, error) {
 	default:
 		return nil, fmt.Errorf("unsupported column type %d", sqltype)
 	}
-	panic("unreachable")
```

Comment:
Drop this change. This panic was inserted for a reason - Go compiler used to complain about "no return statement". So people who use old versions of Go will have problem compiling this code. This code hurts no one.